pylint/checkers/imports.py

```diff
@@ -444,29 +442,11 @@ def __init__(self, linter: Optional["PyLinter"] = None) -> None:
         self._site_packages = self._compute_site_packages()
 
     @staticmethod
-    def _compute_site_packages():
+    def _compute_site_packages() -> Set[str]:
         def _normalized_path(path):
```

Comment:
```suggestion
        def _normalized_path(path: str) -> str:
```

I think `mypy` complains about this with `--strict`.

Reply:
And it should, because it would accept bytes also. But we only get strings from the astroid constant.